docs/cvl/expr.md

```diff
@@ -136,11 +136,16 @@ CVL also adds several useful logical operations:
    For example, the statement `assert initialized => x > 0;` will only report
    counterexamples where `initialized` is true but `x` is not positive.
 
-```{todo}
-Whether implications (and other boolean connectors) are short-circuiting is
-currently undocumented.
-```
-
+ * The short-circuiting behavior of implications (`=>`), bidirectional implications     
+   (`<=>`), and other boolean connectors in CVL mirrors the 
```

Comment:
do we really have short-circuiting for bidirectional implication?

Reply:
I have no idea if we support any short-circuiting, From the request to write that subject i assumed we do but maybe it will be better to delete and let someone familiar with the topic to complete it

Reply:
just remove the bidirection implication. we only do shortcircuiting for impiication, land, lor

Reply:
Done